src/base/static/components/organisms/dashboard/fixed-table.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+
+import makeParsedExpression from "../../../utils/expression/parse";
+import BaseTable from "./base-table";
+
+const fixedTablePropTypes = {
+  data: PropTypes.shape({
+    columns: PropTypes.arrayOf(
+      PropTypes.shape({
+        dataKey: PropTypes.string.isRequired,
+        label: PropTypes.string.isRequired,
+        type: PropTypes.string.isRequired,
+        fractionalWidth: PropTypes.number.isRequired,
+      }).isRequired,
+    ).isRequired,
+    rows: PropTypes.arrayOf(
+      PropTypes.arrayOf(
+        PropTypes.shape({
+          value: PropTypes.array.isRequired,
+          label: PropTypes.string,
+          type: PropTypes.string.isRequired,
+        }).isRequired,
+      ).isRequired,
+    ).isRequired,
+    headers: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
+  }).isRequired,
+  header: PropTypes.string.isRequired,
+  layout: PropTypes.shape({
+    start: PropTypes.number.isRequired,
+    end: PropTypes.number.isRequired,
+    height: PropTypes.string,
+  }).isRequired,
+  stripeColor: PropTypes.string,
+};
```

Comment:
I notice these prop types are being copy/pasted from other components. Instead, I think we should implement them as TS types, define them in a component like BaseTable, and re-use them throughout the code.